File: AC5/src/BubbleSort.java
public class BubbleSort {
  public static int[] Ordena(int[] vetor, int tamanho) {
    int i, fim, aux;

    for (fim = tamanho - 1; fim > 0; --fim) {
      for(i = 0; i < fim; ++i) {
        if (vetor[i] > vetor[i + 1]) {
          aux = vetor[i];
          vetor[i] = vetor[i+1];
          vetor[i+1] = aux;
        }
      }
    }

    return vetor;
  }

}

File: OrdenacaoPython/bubble_sort.py
def bubble_sort(vetor):
    fim = len(vetor)

    while fim > 0: 
        i = 0 
        while i < fim -1:
            if vetor[i] > vetor[i+1]:
                auxiliar = vetor[i]
                vetor[i] = vetor[i+1]
                vetor[i+1] = auxiliar
            i += 1
        fim -= 1

    return vetor

if __name__ == "__main__":
    vetor = [5,4,3,2,1] 
    vetor = bubble_sort(vetor)

    print(vetor)
